deepchem/models/torch_models/cgcnn.py

```diff
@@ -318,8 +341,14 @@ def __init__(self,
     kwargs: Dict
       This class accepts all the keyword arguments from TorchModel.
     """
-    model = CGCNN(in_node_dim, hidden_node_dim, in_edge_dim, num_conv,
+    if kwargs.get('use_openvino', False):
+      model = CGCNN_OV(in_node_dim, hidden_node_dim, in_edge_dim, num_conv,
                   predictor_hidden_feats, n_tasks, mode, n_classes)
+      self.gcgnn = model
+    else:
+      model = CGCNN(in_node_dim, hidden_node_dim, in_edge_dim, num_conv,
+                predictor_hidden_feats, n_tasks, mode, n_classes)
```

Comment:
Is that possible now to wrap a model right before ONNX export? To keep this file unchanged

Reply:
Please restore alignment